src/Drupal/DrupalExtension/Context/MinkContext.php

```diff
@@ -535,6 +535,25 @@ public function assertNotRegionText($text, $region)
         }
     }
 
+  /**
+   * @Then I should not see ( the text) :text in the :region( region) or no such region
+   *
+   * @throws \RuntimeException
+   *   If region or text within it cannot be found.
+   */
+  public function assertNotRegionTextOrNoSuchRegion($text, $region): void {
+    $regionObj = $this->getSession()->getPage()->find('region', $region);
+    if (empty($regionObj)) {
+      return;
+    }
+    //Not find the text within the region.
+    $regionText = $regionObj->getText();
+    if (strpos($regionText, $text) === FALSE) {
+      return;
+    }
```

Comment:
@kumikoono This can be:

```suggestion
    //Not find the text within the region.
    $regionText = !empty($regionObj) ? $regionObj->getText() : '';
    if (strpos($regionText, $text) === FALSE) {
      return;
    }
```